src/products.js

```diff
@@ -0,0 +1,62 @@
+import renderCrystal from './render-crystal.js';
+import store from './data/store.js';
+
+const crystalList = document.getElementById('crystals');
+const crystals = store.getProducts();
```

Comment:
I recommend moving lines 5 - 10 down to line 24 to keep your file structure consistent.